scripts/tags/exturl.js

```diff
@@ -4,8 +4,11 @@
 
 'use strict';
 
-var util = require('hexo-util');
+/*jshint camelcase: false */
+var util = require(hexo.base_dir + 'node_modules/hexo/node_modules/hexo-util');
```

Comment:
My version at test it:
```
hexo-cli: 1.0.2
os: Linux 2.6.32-642.4.2.el6.x86_64 linux x64
http_parser: 1.2
node: 0.10.48
v8: 3.14.5.11
ares: 1.9.0-DEV
uv: 0.10.34
zlib: 1.2.3
modules: 11
openssl: 1.0.1e-fips
```
And `hexo-util` was not found, yes. But if was defined only variable (was in 1st my version, before this fix).
In this fix, there is:
1. `hexo.base_dir` - gloabal hexo variable ABSOLUTE path to root dir (where is source, public, node_modules, etc).
2. 'node_modules/hexo/node_modules/hexo-util' - is second path of string to enforce use internal `hexo-util` directory with all needed modules into this.
If Hexo in root dir in linux, for example, expected path something like this:
`/root/hexo/node_modules/hexo/node_modules/hexo-util` - this will be used, not `/root/hexo/node_modules/hexo-util`.
U can test it by uncomment `//console.log(util);` string at `scripts/tags/exturl.js` file.

> npm lower than version 3 will encounter the error hexo-util module not found.
hexo-util module is present in internal Hexo folder anyway, no matter what version NPM u have. So, with that fix module will use path to the internal module and will work on all versions and all platforms, i think.